extensions-core/avro-extensions/src/main/java/org/apache/druid/data/input/avro/AvroFlattenerMaker.java

```diff
@@ -173,6 +175,20 @@ private Object transformValue(final Object field)
       } else {
         return ((GenericFixed) field).bytes();
       }
+    } else if (field instanceof Map) {
```

Comment:
I wonder if this should use like `if (avroJsonProvider.isArray(field)) ...` and`if (avroJsonProvider.isMap(field)) ... `
There are also methods like `avroJsonProvider.length`, `avroJsonProvider.getArrayIndex`, `avroJsonProvider.getPropertyKeys`, `avroJsonProvider.getMapValue` etc. which could be used to extract stuff, though the array one doesn't seem that useful since it just handles `List` anyway.